minecraft_servers/swacrew/manifest.json

```diff
@@ -0,0 +1,68 @@
+{
+  "server_name": "swacrew",
+  "nice_name": "Swacrew.net",
+  "direct_ip": "swacrew.net",
+  "server_wildcards": [
+    "%.swacrew.net",
+    "%.swacrew.de"
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://swacrew.net",
+    "web_shop": "https://shop.swacrew.net",
+    "twitter": "SwacrewNET",
+    "tiktok": "swacrew",
+    "youtube": "https://www.youtube.com/swacrewnet",
+    "discord": "https://discord.gg/JgFFcVRKed",
+    "teamspeak": "Swacrew.net"
+  },
+  "gamemodes": {
+    "eventserver": {
+      "name": "Eventserver",
+      "url": "https://www.swacrew.net",
```

Comment:
Remove URL parameter in gamemodes (all!)

Reply:
done